fiorella/src/sections/category-products-list.liquid

```diff
@@ -0,0 +1,49 @@
+{%- if section.blocks.size > 0 -%}
```

Comment:
update this file with changes from `https://github.com/magma-labs/shopify-themes/pull/75`